charts/zookeeper/values.yaml

```diff
@@ -1,15 +1,26 @@
 replicas: 3
 
 image:
-  repository: pravega/zookeeper
-  tag: 0.2.9
+  repository: devops-repo.isus.emc.com:8116/nautilus/zookeeper
```

Comment:
Looks like you changed this by mistake, Please revert this change.

Reply:
Reverted